pokedex/src/components/bigCard/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconRuler from '../../assets/ruler.svg';
+import iconWeight from '../../assets/weightScale.svg';
+import imageBackPoke from '../../assets/pokeball-transparent.svg';
+import iconArrowLeft from '../../assets/arrow-left.svg';
+import styles from './styles.module.scss';
+
+function BigCardPokemon({
+  onClick,
+  name,
+  order,
+  src,
+  weigth,
+  height,
+  moves,
+  hp,
+  atk,
+  def,
+  stack,
+  sdef,
+  spd,
+}) {
+  return (
+    <div className={styles.containBigCard} onClick={onClick}>
+      <div className={styles.containHeader}>
+        <img src={iconArrowLeft} alt='arrow left' />
+        <h2>{name} pokemon</h2>
+        <p>#{order}</p>
+      </div>
+      <img
+        className={styles.pokeball}
+        src={imageBackPoke}
+        alt='pokeball transparent'
+      />
+      <img className={styles.pokemonImage} src={src} alt={name} />
+      <div className={styles.containInfo}>
+          <p>About</p>
+        <div>
+          <div>
+            <img src={iconWeight} alt='weight scale' />
+            <p>{weigth}</p>
```

Comment:
Para estos casos podrías utilizar span en vez de p 
y :P
```suggestion
            <p>{weight}</p>
```